contrib/docker/Dockerfile

```diff
@@ -7,8 +7,12 @@
 ###############
 
 # Basic Go environment with git, SSL CA certs, and upx.
-# golang:1.17.1-alpine (linux/amd64)
-FROM golang@sha256:13919fb9091f6667cb375d5fdf016ecd6d3a5d5995603000d422b04583de4ef9 AS builder
+# The image below is golang:1.18.0-alpine (linux/amd64)
```

Comment:
I know it wasn't specified before, but I think we should probably specify the alpine version too since the `-alpine` part is a stand in for the latest version and thus changes as new alpine versions are released.  In other words, if a new alpine version is released, anyone attempting to verify the hash via `golang:1.18.0-alpine` won't match.  I've verified the hash is correct for `golang:1.18.0-alpine3.15`.